research/2019-12-16_searching_with_the_calm_api.md

```diff
@@ -0,0 +1,241 @@
+# Searching with the CALM API
+
+### 16 December 2019, AWLC
```

Comment:
`AWLC`?

Reply:
My initials

Because research notes are a point-in-time snapshot rather than a living thing, I thought it might be useful to record who wrote them – although all the info *should* be here, knowing who to ask is probably useful.

I'll replace it with a full name.